slices/libatomic1.yaml

```diff
@@ -0,0 +1,10 @@
+package: libatomic1
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      # Although gcc-12-base is a package dependency, it is not needed for this
+      # libs slice and is thus omitted.
```

Comment:
I think the `gcc-12-base` dependency is there because the symlink points to its copyright file.

```
lrwxrwxrwx root/root         0 2023-05-13 07:33 ./usr/share/doc/libatomic1 -> gcc-12-base
```

Based on the reply I sent above, I think we can remove this comment. :)

Reply:
done